webrender/src/tiling.rs

```diff
@@ -1559,6 +1563,7 @@ impl StackingContext {
             isolation,
             is_page_root,
             is_visible: false,
+            is_backface_visible: is_backface_visible,
```

Comment:
nit: the greatness of rust allows you to write just `is_backface_visible,` now, DRY